### PR TITLE
Move AnalyzerPowerPack over

### PR DESCRIPTION
**Pull request by tmeschter**

### CODE REVIEW

**Comment by tmeschter** [comment]
@genlu @srivatsn @mavasani Could you take a look, please?

**Comment by genlu** [comment]
@tmeschter Will all these analyzers be moved to FxCop\XXX.Analyzers projects eventually?

File: src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+        {
+            return new CA1008DiagnosticAnalyzer();
+        }
+
+        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
+        {
+            return new CA1008DiagnosticAnalyzer();
+        }
+
+        [WorkItem(836193, "DevDiv")]
```
Comment (genlu) [inline src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs]: Are there corresponding issues on GitHub for those DevDiv WorkItems?

File: src/AnalyzerPowerPack/Test/AnalyzerPowerPack.Test.csproj [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Interoperability\" />
```
Comment (genlu) [inline src/AnalyzerPowerPack/Test/AnalyzerPowerPack.Test.csproj]: @tmeschter There's no `Interoperability` folder.

File: src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs (hunk 1 above)
Comment (tmeschter) [inline src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs]: @genlu No, these issues have been fixed already. We retain the bug number to provide context.

**Comment by tmeschter** [comment]
@genlu No, these analyzers implement FxCop rules that are more related to the languages than specific APIs. The other FxCop rules are being reorganized around the APIs they cover.

File: src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs (hunk 1 above)
Comment (genlu) [inline src/AnalyzerPowerPack/Test/Design/CA1008Tests.cs]: @tmeschter I see. I will update my PR to retain those WorkItems after this one merged.

**Comment by genlu** [comment]
:+1:

**Comment by tmeschter** [comment]
@dotnet-bot Retest this, please.

**Pull request merged**